docker-compose.yml

```diff
@@ -21,7 +23,26 @@ services:
     volumes:
       - ./docker/init-db.sh:/docker-entrypoint-initdb.d/init-db.sh
       - ./docker/pgdata:/var/lib/postgresql/data
+
   redis:
     image: redis
     ports:
       - '6379:6379'
+
+  web:
+    build:
+      context: .
+      target: production
+      dockerfile: apps/web/docker/Dockerfile
+    # This enables to have this service for testing production build
+    # It will not run when doing docker compose up
+    profiles: [building]
```

Comment:
Docker compose allow to have profiles. So no profile runs when `docker compose up` which is perfect for this case. I only want for the combinience of developing Dockerfile production and run `docker compose build web`